test/index.spec.js

```diff
@@ -234,6 +231,137 @@ describe('discipl-law-reg', () => {
+
+      let verifiedAttestor = await core.verify('agree', claimlink2, [minister.did, persoonA.did, null, 'did:discipl:ephemeral:1234', persoonB.did])
+
+      // The first ssid that is valid and proves the attestation should be returned
+      expect(verifiedAttestor).to.equal(minister.did)
+
+    })
+
+    it('should be able to fill functions of single and multiple facts', async () => {
+      let core = lawReg.getAbundanceService().getCoreAPI()
+      let ssid = await core.newSsid('ephemeral')
+
+      let model = {
+        "acts": [],
+        "facts": [
+          {
+            "explanation": "",
+            "fact": "[belanghebbende]",
+            "function": "[persoon wiens belang rechtstreeks bij een besluit is betrokken]",
+            "reference": "art. 1:2 lid 1 Awb",
+            "version": "2-[19940101]-[jjjjmmdd]",
+            "juriconnect": "jci1.3:c:BWBR0005537&hoofdstuk=1&titeldeel=1.1&artikel=1:2&lid=1&z=2017-03-10&g=2017-03-10",
+            "sourcetext": "{Onder belanghebbende wordt verstaan: degene wiens belang rechtstreeks bij een besluit is betrokken}"
+          },
+          {
+            "explanation": "",
+            "fact": "[toezending besluit aan aanvrager]",
+            "function": "[]",
+            "reference": "art 3:41 lid 1 Awb",
+            "version": "",
+            "juriconnect": "",
+            "sourcetext": ""
+          },
+          {
+            "explanation": "",
+            "fact": "[toezending besluit aan meer belanghebbenden]",
+            "function": "[]",
+            "reference": "art 3:41 lid 1 Awb",
+            "version": "",
+            "juriconnect": "",
+            "sourcetext": ""
+          },
+          {
+            "explanation": "",
+            "fact": "[uitreiking besluit aan aanvrager]",
+            "function": "[]",
+            "reference": "art 3:41 lid 1 Awb",
+            "version": "",
+            "juriconnect": "",
+            "sourcetext": ""
+          },
+          {
+            "explanation": "",
+            "fact": "[uitreiking besluit aan meer belanghebbenden]",
+            "function": "[]",
+            "reference": "art 3:41 lid 1 Awb",
+            "version": "",
+            "juriconnect": "",
+            "sourcetext": ""
+          },
+        ],
+        "duties": []
+      }
+
+      let modelLink = await lawReg.publish(ssid, { ...model, 'model': 'AWB' }, {
+        "[uitreiking besluit aan aanvrager]":
+          "IS:assign:did:discipl:ephemeral:1234",
+        "[toezending besluit aan aanvrager]":
+          "IS:assign:did:discipl:ephemeral:1234"
+      })
+
+      let retrievedModel = await core.get(modelLink, ssid)
+
+      let retrievedFact = await core.get(retrievedModel.data['DISCIPL_FLINT_MODEL'].facts[3]['[uitreiking besluit aan aanvrager]'], ssid)
+
+      retrievedFact = retrievedFact.data['DISCIPL_FLINT_FACT'].function
+      expect(retrievedFact).to.deep.equal('IS:assign:did:discipl:ephemeral:1234')
```

Comment:
The 'assign' no longer plays a role in the IS syntax, so we should remove it everywhere